src/talos/cpg_internal_scripts/talos_stages.py

```diff
@@ -373,16 +373,14 @@ def queue_jobs(self, cohort: targets.Cohort, inputs: stage.StageInput) -> stage.
 
         job.command(f'tar -xzf {hail_batch.get_batch().read_input(clinvar_tar)} -C $BATCH_TMPDIR')
 
-        # read in the massive MT, and unpack it
-        localised_mt = hail_batch.get_batch().read_input(input_mt)
-
-        job.command(f'tar -xf {localised_mt} -C $BATCH_TMPDIR && rm {localised_mt}')
+        # read in the MT using gcloud, directly into batch tmp
+        job.command(f'gcloud storage cp -r {input_mt!s} $BATCH_TMPDIR')
```

Comment:
Lean into GCloud. We have it available to us, and need it for the Annotation Workflow. Using super skinny images and Tar/un-tar'ing MTs was causing slow runtimes